src/components/MDXComponents/EmailSubscriptionForm.tsx

```diff
@@ -0,0 +1,139 @@
+import { Trans, useI18next } from "gatsby-plugin-react-i18next";
+import { useState } from "react";
+import { z } from "zod";
+
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import TextField from "@mui/material/TextField";
+
+import { errorMessage } from "./email-subscription-form.module.css";
+
+type FormType = {
+  email: string;
+  loading: boolean;
+  error: null | "invalidEmail" | "networkError";
+};
+
+// Collect email address that subscribe to TiDB release and send it to the SendGrid API
+export function EmailSubscriptionForm() {
```

Comment:
As the release note subscription feature is disabled in the JA docs, we need to wrap `EmailSubscriptionForm` for rendering in different languages.

```suggestion
export function EmailSubscriptionWrapper() {
  const { language } = useI18next();
  const disabled = language === Locale.ja;
  
  if (disabled) {
  	return null;
  }
  
  return <EmailSubscriptionForm />
}

// Collect email address that subscribe to TiDB release and send it to the SendGrid API
function EmailSubscriptionForm() {
```